### PR TITLE
Add 1stdomains.nz

### PR DESCRIPTION
**Pull request by mrtimp**
This PR is to request the addition of https://1stdomains.nz/ to the 2FA directory.  [1st Domains](https://1stdomains.nz) is New Zealand's largest authorised registrar for .nz domains and operates under the [Voyager Internet](https://voyager.nz) brand of companies. 

Here is a screenshot of enabling 2FA:

screenshot_dt1875

### CODE REVIEW

File: entries/1/1stdomains.nz.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "img": "1stdomains.nz.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.1stdomains.co.nz/afmviewfaq.asp?faqid=95",
+    "keywords": [
+      "domains",
+      "hosting",
+      "email"
+    ]
```
Comment (Carlgo11) [inline entries/1/1stdomains.nz.json:13]:
```suggestion
    ],
    "regions": [
      "nz"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo looked a bit large. Here's a smaller one to use instead:
1stdomains nz

**Comment by Carlgo11** [comment]
Oh it's ranked above 200K on Alexa. Closing this PR.

File: entries/1/1stdomains.nz.json (hunk 1 above)
Comment (mrtimp) [reply entries/1/1stdomains.nz.json:13]: Thanks @Carlgo11 

While we are based in New Zealand we have customers worldwide, would adding the region restrict how we are viewed in the directory?

**Comment by mrtimp** [comment]
@Carlgo11 bummer, we were 170K 90 days ago and are ranked 482 within Alexas NZ ranking :-(

What is considered a sizeable following on social media, we may fit in under that criteria.

**Comment by Carlgo11** [comment]
If by *we* you mean you're part of the company behind 1stdomains.nz then may I suggest a redesign of the website to make it looks less like the 1990s. Some CSS and responsive design could do wonders.

**Comment by mrtimp** [comment]
Yes, the website does look its been trapped in the 90's and is well overdue for a redesign!

Thanks for your time.

**Pull request closed without merging**